hokusai/production.yml

```diff
@@ -77,49 +91,63 @@ metadata:
     service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: "300"
 spec:
   ports:
-  - port: 443
-    protocol: TCP
-    name: https
-    targetPort: 8443
-  - port: 80
-    protocol: TCP
-    name: http
-    targetPort: 8080
+    - port: 443
+      protocol: TCP
+      name: https
+      targetPort: 8443
+    - port: 80
+      protocol: TCP
+      name: http
+      targetPort: 8080
   selector:
     app: horizon
     layer: application
     component: web
   sessionAffinity: None
   type: LoadBalancer
-
----
-apiVersion: batch/v2alpha1
-kind: CronJob
-metadata:
-  name: horizon-refresh-cron
-spec:
-  schedule: "*/10 * * * *"
-  concurrencyPolicy: Forbid
-  jobTemplate:
-    spec:
-      template:
-        spec:
-          containers:
-          - name: horizon-refresh-cron
-            image: 585031190124.dkr.ecr.us-east-1.amazonaws.com/horizon:production
-            args: ["bundle", "exec", "rake", "cron:refresh_comparisons"]
-            imagePullPolicy: Always
-            envFrom:
-            - configMapRef:
-                name: horizon-environment
-          restartPolicy: Never
-          affinity:
-            nodeAffinity:
-              preferredDuringSchedulingIgnoredDuringExecution:
-              - weight: 1
-                preference:
-                  matchExpressions:
-                  - key: tier
-                    operator: In
-                    values:
-                    - background
+  loadBalancerSourceRanges:
+    - 173.245.48.0/20
+    - 103.21.244.0/22
+    - 103.22.200.0/22
+    - 103.31.4.0/22
+    - 141.101.64.0/18
+    - 108.162.192.0/18
+    - 190.93.240.0/20
+    - 188.114.96.0/20
+    - 197.234.240.0/22
+    - 198.41.128.0/17
+    - 162.158.0.0/15
+    - 104.16.0.0/12
+    - 172.64.0.0/13
+    - 131.0.72.0/22
+# ---
```

Comment:
Still commented here?

Reply:
See the TODO in the description -- we currently run these jobs in staging, but once we turn production on, we want the jobs to run in production only.